### ISSUE TITLE
"Date Played" Sort Not Working for TV Series in Jellyfin Roku Client

### ISSUE THREAD

**Issue by johnpc** [jellyfin-archive/jellyfin-roku-legacy#2021]
## Software Versions

- Jellyfin Server Version:  10.10.0
- Roku Client Version:  2.2.1

## Describe the bug

The "Date Played" sort option in the Jellyfin Roku client is not functioning correctly for TV series. While this sort option works as expected for movies, it does not appear to have any effect when applied to TV series. The "Date Played" sort is actually much more useful for TV series, since I want to continue watching TV series that I've been playing recently, whereas I'm unlikely to want to re-watch a movie I just saw.

## How To Reproduce

1. Open the Jellyfin Roku client
2. Navigate to a TV series library
3. Select the sort option (using *) and choose "Date Played"
4. Observe that the TV series order does not reflect the watch date

## Expected behavior

the TV series order should surface series that have been watched recently to the top

## Logs

N/A

## Screenshots

N/A

## Connection Information

- Is server local or remote? *local*
- Is server connection HTTP or HTTPS? *https*

## Additional context

N/A

**Comment by nielsvanvelzen** [comment]
For my dear Roku developer friends:

ATV issue: https://github.com/jellyfin/jellyfin-androidtv/issues/4160
ATV fix: https://github.com/jellyfin/jellyfin-androidtv/pull/4168

TL;DR: TV Shows need a different SortBy value
